src/backend/nodes/cypher_readfuncs.c

```diff
@@ -212,7 +212,7 @@ void read_cypher_target_node(struct ExtensibleNode *node)
     READ_LOCALS(cypher_target_node);
 
     READ_CHAR_FIELD(type);
-    READ_UINT_FIELD(flags);
+    READ_INT_FIELD(flags);
```

Comment:
Why were these changed back to INT from UINT. They were purposely changed to UINT in the master.